api/android/README.md

```diff
@@ -235,3 +235,43 @@ You can find the result in ```$NNSTREAMER_ROOT/android_lib```.
 $ cd $NNSTREAMER_ROOT
 $ bash ./api/android/build-android-lib.sh --run_test=yes
 ```
+
+### Using TensorFlow Lite NNAPI Delegate
+
+Since Android 10, all applications are in scoped storage and the `getExternalStorageDirectory()` is deprecated.
+There are some workarounds to use files with external path (e.g. set `requestLegacyExternalStorage` true in app's manifest file). With TensorFlow Lite model files in external storage, however, all workarounds do not work when an application attempts to access the NNAPI, and TensorFlow Lite fails to use available backend like GPU, DSP, or NPU.
+
+If you want to use NNAPI delegate, you should provide the model file from internal storage. You may copy the model files into app-specific directory or cache directory. Or let your application access model files reside in the asset directory.
+
+A Java Example:
+
+```java
+...
+...
+
+String root = Environment.getExternalStorageDirectory().getAbsolutePath();
```

Comment:
Let's consider two methods (do not use instrument test method here)

File modelFromCacheDir(Context context) {
...
}

File modelFromFilesDir(Context context) {
}

Reply:
That would be more clear. I updated it!